src/storage/src/store.rs

```diff
@@ -602,7 +602,6 @@ impl Eq for OpConsistencyLevel {}
 
 impl OpConsistencyLevel {
     pub fn update(&mut self, new_level: &OpConsistencyLevel) {
-        assert_ne!(self, new_level);
```

Comment:
Any reason to disable this check? It's better to do the equality check in state table and remain the same interface to storage, and the equality check is already in `commit_may_switch_consistent_op`.

Reply:
You're right.